### PR TITLE
RavenDB-14076 - Could not find a part of the path

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

File: test/SlowTests/Server/Documents/PeriodicBackup/Retention.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public async Task can_delete_backups_by_date(int backupAgeInSeconds, int numberO
             {
                 BackupConfigurationHelper.SkipMinimumBackupAgeToKeepValidation = true;
 
-                var backupPath = NewDataPath(suffix: "BackupFolder");
+                var backupPath = NewDataPath(suffix: "BackupFolder") + "/Backups/Admin\\..\\..";
```
Comment (ayende) [inline test/SlowTests/Server/Documents/PeriodicBackup/Retention.cs]: Pretty sure that this won't work on Linux, no?

Comment (ppekrol) [reply test/SlowTests/Server/Documents/PeriodicBackup/Retention.cs]: Should work. `PathUtil` will normalize it

File: src/Raven.Server/Documents/PeriodicBackup/Retention/LocalRetentionPolicyRunner.cs [hunk 2]
```diff
@@ -16,7 +17,7 @@ public class LocalRetentionPolicyRunner : RetentionPolicyRunnerBase
         public LocalRetentionPolicyRunner(RetentionPolicyBaseParameters parameters, string folderPath)
             : base(parameters)
         {
-            _folderPath = folderPath;
+            _folderPath = PathUtil.ToFullPath(folderPath, null);
```
Comment (ppekrol) [inline src/Raven.Server/Documents/PeriodicBackup/Retention/LocalRetentionPolicyRunner.cs]: We should use `PathSetting`

Comment (grisha-kotler) [reply src/Raven.Server/Documents/PeriodicBackup/Retention/LocalRetentionPolicyRunner.cs]: done

**Comment by ppekrol** [comment]
@grisha-kotler can you check failing tests?

**Comment by ppekrol** [comment]
Let's reopen when fixed.

**Pull request closed without merging**